packages/kbn-eslint-import-resolver-kibana/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@kbn/eslint-import-resolver-kibana",
+  "description": "eslint-plugin-import resolver for Kibana",
+  "private": true,
+  "version": "2.0.0",
+  "main": "index.js",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elastic/kibana/tree/master/packages/kbn-eslint-import-resolver-kibana"
+  },
+  "dependencies": {
+    "debug": "^2.6.6",
+    "eslint-import-resolver-node": "^0.3.0",
+    "eslint-import-resolver-webpack": "^0.8.1",
```

Comment:
Looks like there's a warning when installing this dep:

```
unmet peer dependency "eslint-plugin-import@>=1.4.0
```

Full output:

```
20:54:35 Installing dependencies in [@kbn/eslint-import-resolver-kibana]:
20:54:35 
20:54:35 [1/4] Resolving packages...
20:54:35 [2/4] Fetching packages...
20:54:36 info fsevents@1.1.3: The platform "linux" is incompatible with this module.
20:54:36 info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
20:54:36 [3/4] Linking dependencies...
20:54:36 warning " > eslint-import-resolver-webpack@0.8.4" has unmet peer dependency "eslint-plugin-import@>=1.4.0".
20:54:37 [4/4] Building fresh packages..```

Reply:
Yeah, I can't think of any way we could prevent this without unnecessarily installing `eslint-plugin-import` and `eslint` as dependencies of the resolver (since `eslint` is a peer dep of `eslint-plugin-import`).

This is one of the errors that I hope yarn ignores if we only ran `yarn` in the top-level dependencies that link and not in the individual packages.